### DISCUSSION TITLE
How can turn on Compilation Mode?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [windicss/vite-plugin-windicss#119] [Q&A]
Hi, i am trying use Svelete kit with vite-plugin-windicss. I have config file `svelte.config.cjs` like this
```
const node = require('@sveltejs/adapter-node');
const pkg = require('./package.json');

/** @type {import('@sveltejs/kit').Config} */
module.exports = {
	kit: {
		// By default, `npm run build` will create a standard Node app.
		// You can create optimized builds for different platforms by
		// specifying a different adapter
		adapter: node(),

		// hydrate the <div id="svelte"> element in src/app.html
		target: '#svelte',

		vite: () => ({
			ssr: {
				noExternal: Object.keys(pkg.dependencies || {})
			},
			plugins: [
				require('vite-plugin-windicss').default({
					prefixer: true,
					prefix: 'tw-',  // this prefix not work too :c
				})
			]
		})
	}
};

```

Can you show me how to turn on Compilation Mode? THANK YOU 
https://windicss.org/guide/modes.html#interpretation-mode

**Comment by antfu** [comment]
The vite plugin does not support compilation mode yet, you can try https://github.com/windicss/svelte-windicss-preprocess instead

**Reply by IRediTOTO** [reply]
ok, thank you. But this is a good feature, right ? :) I will wait for this. 
I am trying to use Svelte-kit, svelte-windicss-preprocess  will not work with it.
